sdk/communication/communication-call-automation/src/models/options.ts

```diff
@@ -218,6 +218,8 @@ export interface PlayOptions extends OperationOptions {
    * This setup is per-action. If this is not set, the default callback URI set by CreateCall/AnswerCall will be used.
    */
   operationCallbackUrl?: string;
+  /** If set play can barge into other existing queued-up/currently-processing requests. */
+  interruptCallMediaOperation?: boolean;
```

Comment:
Now, this is used in `PlayTo` as well. This means that Contoso could pass a value, even if this is not supported. 
I understand that the value is going to be ignored, but Contoso may have an expectation of this working even if it is not supported.
Please, correct that.

Reply:
updated it.